apps/designer/app/shared/router-utils/canvas-request-params.test.ts

```diff
@@ -0,0 +1,91 @@
+import { getCanvasRequestParams } from "./canvas-request-params";
+
+const makeRequest = (url: string, headers: { [key: string]: string }) => ({
+  url,
+  headers: {
```

Comment:
isn't headers just a Map instance?

Reply:
I don't think so. In typescript it's defined separately from Map.